src/services/youtube.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+
+export const baseURL = 'https://www.googleapis.com/youtube/v3';
+
+const service = () => {
+  const apiKey = 'AIzaSyCxpoupiS2E_fiFWXR5mfOTG0JNVS_4T1A';
+  const instance = axios.create({
+    baseURL,
+    timeout: 1500,
```

Comment:
This is cool, I like it, you have everything you need from the youtube API grouped, and it is easy to read and extend